### DISCUSSION TITLE
thoughts on snap support

### DISCUSSION THREAD

**Discussion by Fuseteam** [SuperTux/supertux#3065] [General]
i found there is a snap floating around in the snapstore: https://snapcraft.io/supertux which hasn't been updated in 2 years
i've reached out to the packager and they seem to be open to working together https://github.com/lotharsm/supertux-snapcraft/issues/3
there is also a path to becoming a verified publisher and taking over the entry in the snapstore

sidenote: ubuntu touch is also gaining snap support so this could potentially also be beneficial in that front

**Comment by lotharsm** [comment]
Hi all, I currently have very little time to properly take care of the Snap, but I am absolutely willing to help getting it to upstream or get an update out in the near future - I just can't promise an exact time frame yet.

**Comment by Fuseteam** [comment]
not sure why this looks closed :thinking:

**Reply by Fuseteam** [reply]
thanks for reopening
